src/server/db_slice.h

```diff
@@ -605,7 +609,7 @@ template <typename T> void DbSlice::IteratorT<T>::LaunderIfNeeded() const {
   // falls into a valid range.
   uint64_t current_epoch = util::fb2::FiberSwitchEpoch();
   if (current_epoch != fiber_epoch_) {
-    if (it_->first != key_.view()) {
+    if (!it_.IsOccupied() || it_->first != key_.view()) {
```

Comment:
now you can remove the above comment as IsOccupied will check if we still fall into valid range